### PR TITLE
[zh-cn]: update the translation of Number.isSafeInteger()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15736

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md [hunk 1]
```diff
@@ -5,53 +5,53 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
...
 
-一个{{jsxref("Boolean", "布尔值")}} 表示给定的值是否是一个安全整数（safe integer）。
+如果给定的值是一个安全整数，则返回布尔值 `true`。否则返回 `false`。
+
+## 描述
+
+安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是一个具有以下特征的整数：
+
+- 可以精确地表示为 IEEE-754 双精度数，并且
+- 其 IEEE-754 表示形式不能是舍入任何其他整数以适合 IEEE-754 表示形式的结果。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:32]:
```suggestion
- 其 IEEE-754 表示形式不能是舍入任何其他整数以适应 IEEE-754 表示形式的结果。
```

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:32]:
```suggestion
- 其 IEEE-754 表示形式不能是舍入任何其他整数以适应 IEEE-754 表示形式的结果。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md [hunk 2]
```diff
@@ -5,53 +5,53 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
...
+  - : 要测试是否为安全整数的值。
 
 ### 返回值
 
-一个{{jsxref("Boolean", "布尔值")}} 表示给定的值是否是一个安全整数（safe integer）。
+如果给定的值是一个安全整数，则返回布尔值 `true`。否则返回 `false`。
+
+## 描述
+
+安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是一个具有以下特征的整数：
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:29]:
```suggestion
安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是一个符合以下条件的整数：
```

File: files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md [hunk 3]
```diff
@@ -5,53 +5,53 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
...
+
+- 可以精确地表示为 IEEE-754 双精度数，并且
+- 其 IEEE-754 表示形式不能是舍入任何其他整数以适合 IEEE-754 表示形式的结果。
+
+例如，2<sup>53</sup> - 1 是安全整数：它可以精确地表示，并且在任何 IEEE-754 舍入模式下都不会舍入到它。相比之下，2<sup>53</sup> *不是*安全整数：它可以精确地表示在 IEEE-754 中，但整数 2<sup>53</sup> + 1 不能直接表示在 IEEE-754 中，而是采用舍入到最接近整数和舍入到零时会舍入到 2<sup>53</sup>。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:34]:
```suggestion
例如，2<sup>53</sup> - 1 是一个安全整数：它可以精确地表示，并且在任何 IEEE-754 舍入模式下都没有其他整数舍入结果为该整数。相比之下，2<sup>53</sup> *不是*一个安全整数：它可以在 IEEE-754 中精确地表示，但整数 2<sup>53</sup> + 1 不能直接在 IEEE-754 中表示，而是采用就近舍入和向零舍入时，会舍入到 2<sup>53</sup>。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md [hunk 4]
```diff
@@ -5,53 +5,53 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
...
+## 描述
+
+安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是一个符合以下条件的整数：
+
+- 可以精确地表示为 IEEE-754 双精度数，并且
+- 其 IEEE-754 表示形式不能是舍入任何其他整数以适应 IEEE-754 表示形式的结果。
+
+例如，2<sup>53</sup> - 1 是一个安全整数：它可以精确地表示，并且在任何 IEEE-754 舍入模式下都没有其他整数舍入结果为该整数。相比之下，2<sup>53</sup> *不是*一个安全整数：它可以在 IEEE-754 中精确地表示，但整数 2<sup>53</sup> + 1 不能直接在 IEEE-754 中表示，而是采用就近舍入和向零舍入时，会舍入到 2<sup>53</sup>。
+
+处理比大约 9 千万亿左右的值时，需要使用[任意精度算术库](https://zh.wikipedia.org/wiki/高精度计算)来实现完全精确性。有关浮点数表示的更多信息，请参阅[程序员需要了解的浮点数算术知识](https://floating-point-gui.de/)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:36]:
```suggestion
处理大约 9 千万亿左右的值时，需要使用[任意精度算术库](https://zh.wikipedia.org/wiki/高精度计算)来实现完全精确性。有关浮点数表示的更多信息，请参阅[程序员需要了解的浮点数算术知识](https://floating-point-gui.de/)。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md [hunk 5]
```diff
@@ -5,53 +5,53 @@ slug: Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger
...
+  - : 要测试是否为安全整数的值。
 
 ### 返回值
 
-一个{{jsxref("Boolean", "布尔值")}} 表示给定的值是否是一个安全整数（safe integer）。
+如果给定的值是一个安全整数，则返回布尔值 `true`。否则返回 `false`。
+
+## 描述
+
+安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是一个符合以下条件的整数：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/issafeinteger/index.md:29]: 并不是每一处都适合加“量词”：安全整数不止一个

```suggestion
安全整数由 -(2<sup>53</sup> - 1) 到 2<sup>53</sup> - 1 的所有整数组成，包含（±9,007,199,254,740,991）。安全整数是符合以下条件的整数：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**